tests/example-custom-class/gradlew

```diff
@@ -0,0 +1,183 @@
+#!/usr/bin/env sh
```

Comment:
Is there any particular reason this uses sh and not bash?

Reply:
🤷 `gradlew` is a wrapper file generated by Gradle; I just copied and pasted the existing one from other directories.
But your question got me wondering if we need any gradle files in the `tests/` subfolders - as the test runner uses Maven, not Gradle. Let me try removing the wrappers with `build.gradle` files - it might be just some remaining artifacts from earlier development.

Reply:
I think it indeed is:
- https://github.com/artamonovkirill/groovy-test-runner/commit/2b536e54456a41f0f67279a18cc909a82fe7aef1 added the gradle files
- https://github.com/artamonovkirill/groovy-test-runner/commit/2b536e54456a41f0f67279a18cc909a82fe7aef1 later switched to maven

Reply:
Tested locally without gradle files - it works, as expected; push the commit to this PR